routes/api/decks.js

```diff
@@ -37,19 +37,24 @@ router.get('/', userInfo, asyncHandler(async (req, res, next) => {
     limit: 10,
     offset,
     include: [{
-      model: Category
+      model: Category,
+      attributes: ["label"]
     },
     {
       model: Card,
-      attributes: ["id"]
+      attributes: ["id"] //would prefer to replace this with a count
```

Comment:
I think you can use `Sequelize.fn("COUNT", , Sequelize.col("yourTableColumn")` to add a `cardCount` attribute, like [this](https://stackoverflow.com/questions/37817808/counting-associated-entries-with-sequelize)!

```js
attributes: { 
    include: [[Sequelize.fn("COUNT", Sequelize.col("sensors.id")), "sensorCount"]] 
},
```